besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -715,6 +715,12 @@ static class PrivacyOptionGroup {
         names = {"--privacy-flexible-groups-enabled"},
         description = "Enable flexible privacy groups (default: ${DEFAULT-VALUE})")
     private final Boolean isFlexiblePrivacyGroupsEnabled = false;
+
+    @Option(
+        names = {"--privacy-nonce-incrementation-enabled"},
```

Comment:
What about renaming that to "--privacy-always-increment-nonce" ?
The original name does not really say what this is doing, as we are incrementing the nonce when the tx is successful, just not when it is invalid or not failed.
You might have a better idea for a name :-)